### ISSUE TITLE
Potentially leave User Region alone for Europe Mode

### ISSUE THREAD

**Issue by thepwrtank18** [rcmaehl/MSEdgeRedirect#376]
### Preflight Checklist

- [ ] Running `microsoft-edge:https://google.com` redirects successfully
- [ ] Microsoft Edge is selected in any "How do you want to open this?" box

### Install Type

New Install

### Install Mode

Europe Mode

### Steps to reproduce

1. Enable the two ViveTool IDs needed
2. Start MSEdgeRedirect 0.7.5.3
3. Go to the Europe Mode selection
4. Set EEA country to Ireland
5. Set All to Selected EEA Contry
6. Click Save
7. Restart
8. Go back to MSEdgeRedirect, go to Europe Mode selection
9. Machine Region/Default Region is corrected (in Europe Mode), user region is not

![image](https://github.com/rcmaehl/MSEdgeRedirect/assets/57580668/398b5e39-0ff5-4824-bca6-3ad109da01a0)

### ✔️ Expected Behavior

All of the values would match up.

### ❌ Actual Behavior

The User Region is still in the United States.

### Microsoft Windows version

23H2 Build 22631.2861

**Comment by rcmaehl** [comment]
Yeah, this seems to keep getting reset. I may remove it (if it still has no effect) in March once Microsoft adds in all their EEA feature promises.

**Comment by turnipjs** [comment]
Post-March bump in case you wanted to be reminded.
